packages/search-adapter/src/empathy/mappers/request/params/empathy-request-filters-solr-syntax.mapper.ts

```diff
@@ -29,8 +31,8 @@ export class EmpathyRequestFiltersSolrSyntaxMapper implements Mapper<Filter[], s
   }
 
   private composeFilters(filters: Filter[]): string[] {
-    if (this.isArrayOfBooleanFilters(filters)) {
-      return filters.map(filter => filter.value);
+    if (this.isArrayOfBooleanFilters(filters) || this.isArrayOfRawFilters(filters)) {
+      return filters.map(filter => `${filter.id}`);
```

Comment:
Just for my info, if we add more types in the future we can just treat the EditableNumberRangeFilterList as a specific case and the rest in an else statement ? Or are there a type of filters which don't need to be mapped here ?